### ISSUE TITLE
Search for generators doesn't work

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#4788]
We forgot to add generators (and possible search synonyms) to categories.txt, and the search for them doesn't work. See these translations in data/strings/types_strings.txt
- [type.power.generator]
- [type.power.generator.solar]
- [type.power.generator.wind]
- [type.power.generator.gas]
- [type.power.generator.hydro]

It raises again the importance of the search refactoring when all categories will be searched automatically from types_strings.txt, only synonyms and exceptions will be specified in categories.txt https://github.com/organicmaps/organicmaps/issues/1795

**Comment by Troll-of-Jom** [comment]
Hey!! Can you assign me this issue

**Comment by biodranik** [comment]
No. We do not assign newcomers to issues. If you want to fix it, please do so without an assignment.

**Comment by N-45div** [comment]
@biodranik sir i have made the changes please do review it
regards,
N DIVIJ

**Comment by N-45div** [comment]
@biodranik sir i have pushed the changes please review it
regards,
N DIVIJ
